frontend-react/src/hooks/UseOrganizationReceiversFeed.ts

```diff
@@ -29,9 +34,10 @@ export const useOrganizationReceiversFeed = (): ReceiverFeeds => {
     }, [receivers]);
 
     return {
-        loadingServices: isLoading,
+        loadingServices: isLoading && fetchStatus !== "idle",
         services: receivers || [],
         activeService: active,
         setActiveService: setActive,
+        isDisabled: isLoading && fetchStatus === "idle",
```

Comment:
I really hate that react-query queries don't pass through the `enabled` status as an `isEnabled`.  It feels like such a missed opportunity...especially since we have to grapple with two load states like this 😩

Reply:
So what you're saying is ... `enabled != isEnabled` but `enabled == !isEnabled` 😂

Reply:
Hahahah EXACTLY 😆